iaac/terraform/kubeflow-components/profiles-and-kfam/main.tf

```diff
@@ -0,0 +1,43 @@
+resource "aws_iam_policy" "profile_controller_policy" {
+  name_prefix        = "profile-controller-policy"
+  description = "IAM policy for the kubeflow pipelines profile controller"
+  policy        = "${file("../../../awsconfigs/infra_configs/iam_profile_controller_policy.json")}"
+}
+
+module "irsa" {
+  source            = "github.com/aws-ia/terraform-aws-eks-blueprints//modules/irsa?ref=v4.9.0"
+  kubernetes_namespace = "kubeflow"
+  create_kubernetes_namespace = false
+  create_kubernetes_service_account = false
```

Comment:
false?

Reply:
service account is created by the helm chart

as a workaround i create it below w the necessary annotations for helm to adopt it, but the helm charts should provide the option to pass in irsa role